apache2/waf_logging/waf_log_util_external.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _WAF_LOG_UTIL_EXTERNAL_HEADER
+#define _WAF_LOG_UTIL_EXTERNAL_HEADER
+
+#define WAF_LOG_UTIL_FAILED -1
+#define WAF_LOG_UTIL_SUCCESS 0
+#define WAF_RULESET_PREFIX "/RuleSets/"
+#define WAF_LOG_UTIL_FILE "waf_json.log"
+#define WAF_LOG_UTIL_OPERATION_NAME "ApplicationGatewayFirewall"
```

Comment:
Should we add these as defaults in the proto definition?

Reply:
i think it's better to pull all the waf related dependency out of protobuf.  Protobuf should be a standalone utility for us and we put all the waf related info in the header file we exposed.